config/initializers/004-message_bus.rb

```diff
@@ -111,25 +111,18 @@ def setup_message_bus_env(env)
   ActiveRecord::Base.connection_handler.clear_active_connections!
 end
 
-if Rails.env == "test"
+if Rails.env.test?
   MessageBus.configure(backend: :memory)
 else
   MessageBus.redis_config = GlobalSetting.message_bus_redis_config
 end
 
 MessageBus.backend_instance.max_backlog_size = GlobalSetting.message_bus_max_backlog_size
 
-if SiteSetting.table_exists? && SiteSetting.where(name: ['enable_long_polling', 'long_polling_interval']).exists?
-  Discourse.deprecate("enable_long_polling/long_polling_interval have switched from site settings to global settings. Remove the override from the Site Settings UI, and use a config file or environment variables to set the global settings.", drop_from: '2.9.0')
```

Comment:
Dropped this code as it’s been deprecated for a while and should be dropped with version 2.9

Reply:
Do we have a migration to clean up the rows in the database too?

Reply:
Ah no, I didn’t think about that :thinking: So we should simply remove entries where these names appear?

Reply:
Yup something like https://github.com/discourse/discourse/blob/main/db/migrate/20180127005644_remove_old_ga_site_settings.rb :)

Reply:
Done!

Reply:
Thank you :+1: